packages/isomorphic/README.md

```diff
@@ -0,0 +1,100 @@
+# @xrplf/isomorphic
+
+A collection of isomorphic implementations of crypto and utility functions.
+
+Browser implementations of cryptographic functions use `@noble/hashes` and `crypto` for node .
+
+### Hashes
+
+All hash functions operate similarly to `@noble/hashes` and have the following properties:
+
+- They can be called directly by providing a Uint8Array or string which will be converted into a UInt8Array via UTF-8 encoding (not hex).
+- They all return a UInt8Array.
+
+```
+function hash(message: Uint8Array | string): Uint8Array;
+hash(new Uint8Array([1, 3]));
+hash('string') == hash(new TextEncoder().encode('string'));
+```
+
+All hash functions can be constructed via `hash.create()` method:
+
+- The result is `Hash` subclass instance, which has `update()` and `digest()` methods.
+- `digest()` finalizes the hash and makes it no longer usable
```

Comment:
What does this make no longer usable? (The `Hash` subclass?) - Why is it no longer usable?

Reply:
I think that once you call digest you cant call update again?